### ISSUE TITLE
USWDS - Memorable date: Combobox labels can be confusing for screen reader users.

### ISSUE THREAD

**Issue by jaclinec** [uswds/uswds#5911]
# Summary
 Visually impaired users of screen readers are confused by the announcement of the options in the memorable date combobox because they include a number and a month (i.e. "01 - January"). They sometimes confuse it to mean a month and day (i.e. "January 1st"). It may better meet expectations and lessen confusion to list only the month name without the number.

# Observations
A blind participant tried to choose the month in the memorable date dropdown during usability testing and heard it announce "01 - January" and thought it meant 'January 1st.' He also commented that that was not something he expected and that it should only list the month names with no numbers.

# Affected user groups

- Screen reader users

# Research method
Usability testing with 8 participants with disabilities: 
- 5 visually impaired (3 blind, 2 low vision/legally blind (1 with nystagmus))
- 1 motor impairment
- 1 ADHD
- 1 anxiety and depression

Screen reader software participants used: JAWS, NVDA, and VoiceOver

[Link to public findings report forthcoming...here is the internal [Zebra findings report](https://docs.google.com/document/d/1vbGSh4UNZiseyzdGXa1pJwU_3M4y7HaU3QPkXOa7gsI/edit?usp=sharing)] :lock:

# Next steps

- [ ] Remove the numbers from the options and only display the month names. **Note:** Only do this if it is possible to allow users to still type a name OR a number in the combo box to search for the month (see #5910 ), since it is important to allow users to search by either the name or the number of the month based on zebra usability testing results.

**Comment by amyleadem** [comment]
@jaclinec Is this a duplicate of #5910

**Comment by jaclinec** [comment]
@amyleadem The issues are definitely closely related and are kind of the same now. 

Originally, I created 2 separate issues for 2 separate things: 

- this issue was to simply **remove the numbers from the list and only display month names**. 
- Issue #5910 was originally meant to address the **search functionality** of the memorable date dropdown. I was hoping we could allow users to search with either month name OR number, even if only the name was displayed. That way, for those users who actually type in, say, "11" then November would be selected. I thought it used combo box functionality but now that I know it uses select, pretty sure we can't do that.

Long story short, we can probably close this issue in favor of #5910
